caluma/caluma_workflow/tests/snapshots/snap_test_work_item.py

```diff
@@ -78,162 +78,6 @@
     }
 }
 
-snapshots[
```

Comment:
Why did those snapshots disapper? They don't seem to be related to the change of this PR

Reply:
Huh that's odd. I would assume that the CI would complain about this?
No idea tbh

Reply:
Maybe someone didn't clean them up before? But I'd still investigate this